### DISCUSSION TITLE
Netgear X6 R8000

### DISCUSSION THREAD

**Discussion by nmz787** [SmoothWAN/SmoothWAN#24] [General & Help]
An R8000 image is listed here, present too is the imagebuilder item:
https://downloads.openwrt.org/releases/21.02.1/targets/bcm53xx/generic/

but I then saw there's no R8000 in the smoothWAN repo's `devconfigs` folder. What do I do now?

I've already got an R8000 that has been pretty reliable with freshtomato, and which I've had open-wrt running on before (the image linked above). They seem well powered, and cost reasonable in used markets. They've got multiple USB ports on them.

**Comment by TalalMash** [comment]
Unfortunately the resources on these are very scarce, and no AES acceleration to back it up.
NanoPi R4S is supported in the next release, should cost less, and much more powerful than a Pi 4, the reason for support is due to endless chip shortage with the Pi's, although Pi's are primarily supported due to global availability, not for performance or cost.
Note that OSS for wireless modules are (in majority) not optimized/reliable, stock firmware is recommended for routers/AP when used with SmoothWAN.

**Comment by nmz787** [comment]
Would I still be able to achieve around 30-50 MBPS max? I am not looking for throughput as much as for bonding multiple LTE modems for better latency (and any bandwidth aggregation/increase would be welcome).

**Reply by TalalMash** [reply]
The memory is below the minimum requirment. (512MB)

**Comment by nmz787** [comment]
Is there another common router that would be recommended? I don't quite feel immediate trust for the nanopi r4s.

**Reply by TalalMash** [reply]
Check GL.iNet Brume-W, it has enough power for <100Mbit but with poor Wi-Fi and no USB ports.
https://github.com/TalalMash/SmoothWAN/discussions/19#discussioncomment-2538590
I don't know what your requirments are.
It's difficult to find a Wi-Fi enabled router with decent modules, enough power and designed for OpenWRT.
Xiaomi Wi-Fi 6 and few Asus seems to be the closest with open source firmware and closed source binary blobs for Wi-Fi for optimal performance. Needs patching, no flashable images are possible and it's a bit hacky.
Modding a router/AP with a Pi 4 inside seems to be the only and easiest solution for low footprint AIO setup.

**Reply by nmz787** [reply]
I don't have space concerns, and the X6 R8000 I have running freshtomato right now seems fine in terms of wifi performance all around my house, so I can keep using that as an AP and main router. I need at least 2 USB ports on the smoothWAN load balancer (at least that's what I've been calling it), since I have 2 LTE modems that talk to different cell towers for redundancy. In the future I may get Starlink, if I ever get off their waitlist.
